packages/components/src/utils/config-values.js

```diff
@@ -4,13 +4,41 @@
 import { space } from '../ui/utils/space';
 import { COLORS } from './colors-values';
 
-const CONTROL_HEIGHT = '30px';
+const CONTROL_HEIGHT = '36px';
 const CONTROL_PADDING_X = '12px';
-
+const CONTROL_PROPS = {
+	controlSurfaceColor: COLORS.white,
+	controlTextActiveColor: COLORS.ui.theme,
+	controlPaddingX: CONTROL_PADDING_X,
+	controlPaddingXLarge: `calc(${ CONTROL_PADDING_X } * 1.3334)`,
+	controlPaddingXSmall: `calc(${ CONTROL_PADDING_X } / 1.3334)`,
+	controlBackgroundColor: COLORS.white,
+	controlBorderRadius: '2px',
+	controlBorderColor: COLORS.gray[ 700 ],
+	controlBoxShadow: 'transparent',
+	controlBorderColorHover: COLORS.gray[ 700 ],
+	controlBoxShadowFocus: `0 0 0, 0.5px, ${ COLORS.admin.theme }`,
```

Comment:
This is the value Joen mentions here: https://github.com/WordPress/gutenberg/pull/31937#issuecomment-863105474